src/main/kotlin/com/codescene/jetbrains/components/statusbar/AceStatusBarWidget.kt

```diff
@@ -0,0 +1,67 @@
+package com.codescene.jetbrains.components.statusbar
+
+import com.codescene.jetbrains.CodeSceneIcons.CODESCENE_TW
+import com.codescene.jetbrains.config.global.AceStatus
+import com.codescene.jetbrains.notifier.AceStatusRefreshNotifier
+import com.codescene.jetbrains.services.AceService
+import com.codescene.jetbrains.util.Log
+import com.intellij.openapi.application.ApplicationManager
+import com.intellij.openapi.project.Project
+import com.intellij.openapi.wm.StatusBar
+import com.intellij.openapi.wm.StatusBarWidget
+import com.intellij.openapi.wm.StatusBarWidgetFactory
+import com.intellij.util.Consumer
+import java.awt.event.MouseEvent
+import javax.swing.Icon
+
+class AceStatusBarWidget : StatusBarWidget.IconPresentation, StatusBarWidget {
+    private var statusBar: StatusBar? = null
+    private var value: String = AceService.getInstance().getStatus().value
+
+    override fun ID(): String = "AceStatusBarWidget"
```

Comment:
I guess we could do this here instead

```suggestion
    override fun ID(): String = this::class.simpleName!!
```

to avoid hardcoding the name. Same thing for `AceStatusBarWidgetFactory`'s ID.